cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/spark/SparkProgramRunnerTest.java

```diff
@@ -155,6 +163,24 @@ public void testSparkWithObjectStore() throws Exception {
     prepareInputData();
     runProgram(app, SparkAppUsingObjectStore.CharCountSpecification.class);
     checkOutputData();
+
+    // validate that the table emitted metrics
+    Collection<MetricTimeSeries> metrics =
+      metricStore.query(new MetricDataQuery(
```

Comment:
it's time to have some builders.. :) We have them for Cube, but not for MetricsStore (which was used mostly internally so far). Created https://issues.cask.co/browse/CDAP-3112